front/src/Phaser/Game/GameScene.ts

```diff
@@ -1060,26 +1059,26 @@ ${escapedMessage}
         }
     }
 
-    private initPositionFromLayerName(layerName: string) {
+    private initPositionFromLayerName(layerName: string, startLayerName: string | null) {
         for (const layer of this.gameMap.layersIterator) {
-            if ((layerName === layer.name || layer.name.endsWith('/'+layerName)) && layer.type === 'tilelayer' && (layerName === defaultStartLayerName || this.isStartLayer(layer))) {
-                const startPosition = this.startUser(layer);
-                this.startX = startPosition.x + this.mapFile.tilewidth/2;
-                this.startY = startPosition.y + this.mapFile.tileheight/2;
+            if ((layerName === layer.name || layer.name.endsWith('/' + layerName)) && layer.type === 'tilelayer' && (layerName === defaultStartLayerName || this.isStartLayer(layer))) {
+                const startPosition = this.startUser(layer, startLayerName);
+                this.startX = startPosition.x + this.mapFile.tilewidth / 2;
+                this.startY = startPosition.y + this.mapFile.tileheight / 2;
```

Comment:
Does it work here?

I mean, if a tile with a "start" property is present on any layer that does not satisfy the "if" condition, it would be ignored, right?

Reply:
yes the idea is to have a layer called start nad put all of the tiles in that 🤔 otherwise it would take quite some time to search through al tiles of all layers

Reply:
i mean yes it worked (i only planned to allow tile properties in the "start" layer so as to not have to go through all the data arrays) but i tihnk its not bakwards compatible currently 😅 gimme a sec